### ISSUE TITLE
[Feature Request] - Jukebox mode

### ISSUE THREAD

**Issue by zkvsky** [CappielloAntonio/tempo#222]
## Summary

I'd like to use Jukebox mode of  Subsonic. The client in this mode works as remote controller, for  another box.
## Description

Usually it's implemented under cast button, to pick the streaming destination, with additional option in settings to do this by default (meaning to use the Jukebox mode)
## Use Case

Raspberry Pi with susbonic server [Navidrome] sitting on it with speakers connected to it. Android app functions as remote controller for this "Jukebox"

**Comment by kylian-002** [comment]
I would love to have this as well. My use case is very similar. In my case i would install Tempo on a Eversolo DMP-A6. Due to my music library being too large for SMB/NFS on the eversolo. Would also be great if multiple people could fill up a queue on a jukebox client. Similar how spotify connect works where multiple people can join the session on the device.

**Comment by MRodneyMcKay** [comment]
A better, more useful implementation of this would be openhome dlna casting.
I've made a feature request for this

**Comment by kylian-002** [comment]
> A better, more useful implementation of this would be openhome dlna casting. I've made a feature request for this

Personally I do not like dlna because it often requires the insecure UPnP as well. Also it gets very frustrating quickly to get working if you have things like vlans set up.

**Comment by MRodneyMcKay** [comment]
> insecure UPnP

I have disabled upnp on my router, i run my own upnp server on the LAN side, altough ut may not be that common, I suppose....

You are right, that jukebox is easier if, you set up virtual networks.
jukebox is, in my mind, superior to standard dlna, because the now playing que is owned by the server, so it does not use any resource on the control point. So, from this point of view, jukebox and openhome are almost the same.

**Comment by MRodneyMcKay** [comment]
Yeah, and tempo coupled with navidrome (with jukebox) will be a splendid option, when the new scanner will be ready on navidrome's side, so multivalue tags would be respected.

**Comment by MRodneyMcKay** [comment]
Is this repository abandoned?

**Comment by CappielloAntonio** [comment]
Absolutely not, it's not an abandoned project and I have no intention of abandoning it. Life is simply keeping me away more than expected, but I have no intention of giving up now. Anyway, anyone is free to contribute code, that's the beauty of an open source project.

**Comment by black00019** [comment]
Good Morning @CappielloAntonio 
I love "Tempo", and i have enabled in my selfhosted navidrome jukebox mode, but Tempo not have this feature, but i found only DSub App have.
It's possible developing this feature on Tempo?
Thanks
